src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const reversed = {};
```

Comment:
It's not clear what should be stored in this variable
`reversedRobot` will be better variable name